kafka/consumer-group/consumer_group.go

```diff
@@ -340,7 +352,15 @@ func (cg *ConsumerGroup) topicConsumer(topic string, messages chan<- *sarama.Con
 	// Consume all the assigned partitions
 	var wg sync.WaitGroup
 	for _, pid := range myPartitions {
-
+		//Create PartitionInfo and send it for ingestion through the partition channel
+		//In case of re-balancing this function will be triggered again and the latest information will be sent
+
+		metricName := name + "." + topic + "." + cg.instance.ID + "." + time.Now().Format(time.RFC850)
+		metrics.Reg.Ingest(metrics.Metric{
+			MetricType:  metrics.GAUGE,
+			MetricName:  "partition_owned." + metricName,
```

Comment:
can we build metric once?

Reply:
But the metric Name changes every time

Reply:
if you can see above, metric name is just built before it's getting ingested. how will it change?